content/en/news/lets-celebrate-hugos-5th-birthday/index.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2018-07-04
+title: "Let’s celebrate Hugo’s 5th birthday"
+description: "How a side project became the fastest and one of the most popular framework for building websites."
+categories: [blog]
+author: bep
+---
+
+
+Gopher artwork by [Ashley McNamara](https://github.com/ashleymcnamara/gophers/) (licensed under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)). Inspired by [Renee French](https://reneefrench.blogspot.com/).
+
+Some notable sites.
+
+https://www.smashingmagazine.com/ https://vote.gov/ https://www.netlify.com https://gohugo.io/showcase/letsencrypt/ https://kubernetes.io/
```

Comment:
Should these sites be put as individual list items?

Reply:
I just dumped them in there... bokmarks for later. When I asked for input I meant on a more general level. This is a rough draft that I will finish tomorrow.